src/TemplateProcessor.ts

```diff
@@ -121,7 +121,8 @@ export default class TemplateProcessor {
 
     private tempVars:JsonPointerString[];
 
-
+    /** onInit can be overwritten to prove optional callbacks to be called on initialization */
+    public static onInit = () => {};
```

Comment:
The variable to override cannot be static. If it is static is means we cannot use more than one TemplateProcessor in the same Node process at the same time. In your constructor you can to `this.onInit = ()=>{}` which creates the instance-local onInit variable.

Reply:
yep, my original thinking was that this is a feature for a repl server, so in this way it is a one process at a time indeed. I've changed it to be an instance-local var